src/Attribute/TypeOverride.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CoolBeans\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class TypeOverride
+{
+    public function __construct(public string $type)
```

Comment:
newline, so that the constructor property promotion is more visible eg:

```
public function __construct(
    public string $defaultValue
) {}
```